chain/jsonrpc/Cargo.toml

```diff
@@ -31,6 +32,7 @@ near-jsonrpc-client = { path = "client" }
 near-jsonrpc-primitives = { path = "../jsonrpc-primitives" }
 near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
 near-performance-metrics = { path = "../../utils/near-performance-metrics" }
+near-store = { path = "../../core/store" }
```

Comment:
I don't believe that store is a reasonable dependency for JSON RPC. Is there a way to avoid it?

Reply:
@pmnoxx I unresolved the issue (I believe we might remove it if we move `make_peer_manager` as per my other comment)

Reply:
I removed `near-store` dependecy and moved `make_peer_manager` to near-network ... `test_utils.rs`